package.json

```diff
@@ -61,7 +60,8 @@
     "@istanbuljs/nyc-config-typescript": "^0.1.3",
     "@types/mongodb": "^3.3.6",
     "@types/mongodb-memory-server": "^1.8.0",
-    "ava": "^2.4.0",
+    "@types/tmp": "^0.1.0",
```

Comment:
What is tmp?

Reply:
It's a dependency of `mongodb-memory-server-core`, which in turn is a dependency of `mongodb-memory-server`. The problem is that with the latest version of `mongodb-memory-server`, the TypeScript build breaks without `@types/tmp` explicitly installed. That's a problem on their side (as their deve dependencies don't seem to include `@types/tmp`).

Reply:
OK